### PR TITLE
Fixes for CONTRIB-67

### PR DESCRIPTION
**Pull request by maoo**
Following comments on https://symphonyoss.atlassian.net/browse/CONTRIB-67 , I've wrapped up the changes needed in order to comply with Foundation's legal requirements, see https://symphonyoss.atlassian.net/wiki/display/FM/Legal+Requirements

### CODE REVIEW

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
+# Contributing to unfurl bot
```
Comment (pmonks) [inline .github/CONTRIBUTING.md]: Should this be "SSF Desktop Wrapper" rather than "unfurl bot"?

File: .github/CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+# Contributor License Agreement (CLA)
+A CLA is a document that specifies how a project is allowed to use your
+contribution; they are commonly used in many open source projects.
+
+**_All_ contributions to _all_ projects hosted by the
+[Symphony Software Foundation](https://symphony.foundation/) must be made with
+a [Foundation CLA](https://symphonyoss.atlassian.net/wiki/display/FM/Legal+Requirements#LegalRequirements-ContributorLicenseAgreement)
+in place, and there are [additional legal requirements](https://symphonyoss.atlassian.net/wiki/display/FM/Legal+Requirements) that must also be met.**
+
+As a result, PRs submitted to the bot-unfurl project cannot be accepted until you have a CLA in place with the Foundation.
```
Comment (pmonks) [inline .github/CONTRIBUTING.md]: I believe "bot-unfurl" should be changed to "ssf-desktop-wrapper".

File: .github/CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+# Contributing Issues
+
+## Prerequisites
+
+* [ ] Have you [searched for duplicates](https://github.com/symphonyoss/bot-unfurl/issues?utf8=%E2%9C%93&q=)?  A simple search for exception error messages or a summary of the unexpected behaviour should suffice.
```
Comment (pmonks) [inline .github/CONTRIBUTING.md]: URL needs updating.

File: .github/CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+* [ ] Are you running the [latest release of the unfurl bot](https://github.com/symphonyoss/bot-unfurl/commits/master)?
+
+## Raising an Issue
+* Create your issue [here](https://github.com/symphonyoss/bot-unfurl/issues/new).
```
Comment (pmonks) [inline .github/CONTRIBUTING.md]: URL needs updating.

File: .github/CONTRIBUTING.md [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+in place, and there are [additional legal requirements](https://symphonyoss.atlassian.net/wiki/display/FM/Legal+Requirements) that must also be met.**
+
+As a result, PRs submitted to the bot-unfurl project cannot be accepted until you have a CLA in place with the Foundation.
+
+# Contributing Issues
+
+## Prerequisites
+
+* [ ] Have you [searched for duplicates](https://github.com/symphonyoss/bot-unfurl/issues?utf8=%E2%9C%93&q=)?  A simple search for exception error messages or a summary of the unexpected behaviour should suffice.
+* [ ] Are you running the [latest release of the unfurl bot](https://github.com/symphonyoss/bot-unfurl/commits/master)?
```
Comment (pmonks) [inline .github/CONTRIBUTING.md]: URL needs updating.

File: .github/CONTRIBUTING.md [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+* Please use [Markdown formatting](https://help.github.com/categories/writing-on-github/)
+liberally to assist in readability.
+  * [Code fences](https://help.github.com/articles/creating-and-highlighting-code-blocks/) for exception stack traces and log entries, for example, massively improve readability.
+
+# Contributing Pull Requests (Code & Docs)
+To make review of PRs easier, please:
+
+ * Please make sure your PRs will merge cleanly - PRs that don't are unlikely to be accepted.
+ * For code contributions, follow the existing code layout.
+ * For documentation contributions, follow the general structure, language, and tone of the [existing docs](https://github.com/symphonyoss/bot-unfurl/wiki).
```
Comment (pmonks) [inline .github/CONTRIBUTING.md]: URL needs updating.

File: .github/CONTRIBUTING.md [hunk 7]
```diff
@@ -0,0 +1,41 @@
...
+ * Keep PRs small and cohesive - if you have multiple contributions, please submit them as independent PRs.
+ * Reference issue #s if your PR has anything to do with an issue (even if it doesn't address it).
+ * Minimise non-functional changes (e.g. whitespace shenanigans).
+ * Ensure all new files include a header comment block containing the [Apache License v2.0 and your copyright information](http://www.apache.org/licenses/LICENSE-2.0#apply).
+ * If necessary (e.g. due to 3rd party dependency licensing requirements), update the [NOTICE file](https://github.com/symphonyoss/bot-unfurl/blob/master/NOTICE) with any new attribution or other notices
```
Comment (pmonks) [inline .github/CONTRIBUTING.md]: URL needs updating.

**Comment by maoo** [comment]
Thanks @pmonks for the review! I fixed all the issues you pointed out. the PR is up-to-date now

File: .github/CONTRIBUTING.md [hunk 8]
```diff
@@ -0,0 +1,41 @@
...
+
+## Prerequisites
+
+* [ ] Have you [searched for duplicates](https://github.com/ScottLogic/ssf-desktop-wrapper/issues?utf8=%E2%9C%93&q=)?  A simple search for exception error messages or a summary of the unexpected behaviour should suffice.
+* [ ] Are you running the [latest release of the ssf-desktop-wrapper](https://github.com/ScottLogic/ssf-desktop-wrapper/commits/master)?
+
+## Raising an Issue
+* Create your issue [here](https://github.com/ScottLogic/ssf-desktop-wrapper/issues/new).
+* New issues contain two templates in the description: bug report and enhancement request. Please pick the most appropriate for your issue, and delete the other.
+  * Please also tag the new issue with either "Bug" or "Enhancement".
```
Comment (ColinEberhardt) [inline .github/CONTRIBUTING.md:25]: minor point - unless you have write access to a repo you cannot label issues, which is a bit of a pain. Labeling is unfortunately a triage task!

**Review by ColinEberhardt — APPROVED** [review]

**Pull request merged**